README.md

```diff
@@ -164,14 +164,14 @@ worker_ssh_ingress = "0.0.0.0/0"
 $ terraform output ssh_private_key > generated/instances_id_rsa
 # Retrieve public IP for etcd nodes
 $ terraform output etcd_public_ips
-# Log in as user ubuntu to the Canonical Ubuntu OS
-$ ssh -i `pwd`/generated/instances_id_rsa ubuntu@ETCD_INSTANCE_IP
+# Log in as user opc to the OEL OS
+$ ssh -i `pwd`/generated/instances_id_rsa oel@ETCD_INSTANCE_IP
```

Comment:
user looks weird here  'oel'    should be 'opc'  right.

Reply:
Good eye. Pushed [fix](https://github.com/oracle/terraform-kubernetes-installer/commit/ca08a74c80a0f1d588a21ce7fe8fc5391ea9519e).